### PR TITLE
feat(prisma): upgrade to V4

### PR DESCRIPTION
**Pull request by tomapp**
## Information

| Type                  | Breaking change |
| --------------------- | --------------- |
| Chore | Yes          |

---

I think it require a bit further investigations actually. But according to prisma there's not much changes to deal with.

## Todos

- [ ] Tests
- [ ] Coverage
- [ ] Example
- [ ] Documentation

### CODE REVIEW

**Review by Romakita — APPROVED** [review]

File: packages/orm/prisma/package.json [hunk 1]
```diff
@@ -24,25 +24,25 @@
...
     "@tsed/json-mapper": "6.117.2",
     "@tsed/schema": "6.117.2",
     "@types/change-case": "^2.3.1",
     "@types/pluralize": "0.0.29",
-    "prisma": "^3.10.0"
+    "prisma": "^4.0.0"
   },
   "peerDependencies": {
-    "@prisma/client": "^2 || ^3",
+    "@prisma/client": "4.0.0",
```
Comment (martin31821) [inline packages/orm/prisma/package.json:45]: Hey, since prisma 4.1.1 is out these days, this gives an error. I think this value should be ^4.0.0?

Comment (Romakita) [reply packages/orm/prisma/package.json:45]: Yes, I missed this wrong peerDependencies declaration during the code review. can you make a short PR @martin31821 or @tomapp  please ?

Comment (Romakita) [reply packages/orm/prisma/package.json:45]: it should be >=4.0.0

**Pull request merged**